app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -164,19 +149,48 @@ Set up a Teaching Vacancies job alert to be updated about opportunities througho
 </p>
 
 
-## Find out what salary you can earn
+## Apply for your visa
+
+You can apply for a skilled worker visa once you have been offered a teaching job at a school which can sponsor them. Our [guidance on visas and immigration](https://www.gov.uk/government/publications/teach-in-england-if-you-qualified-outside-the-uk/teach-in-england-if-you-qualified-outside-the-uk#visas-and-immigration) explains how to apply for a skilled worker visa, what salary you will need to earn and other types of visa you may be eligible for. 
+
+You may have a pre-existing right to work in the UK, such as settled or pre-settled status under the [EU Settlement Scheme](https://www.gov.uk/settled-status-eu-citizens-families) or [indefinite leave to remain in the UK](https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk).
+
+[If you're an Irish citizen you do not need a visa](https://www.gov.uk/government/publications/common-travel-area-guidance).
+
+If you’re not sure about your immigration status, you can [check if you need a UK visa](https://www.gov.uk/check-uk-visa) or [contact UK Visas and Immigration for help](https://www.gov.uk/contact-ukvi-inside-outside-uk).
+
+You can also learn more about [bringing your partner and children](https://www.gov.uk/skilled-worker-visa/your-partner-and-children).
+
+## Plan your move to the UK
+
+The following resources may be useful as you plan your move:
+
+* [open a digital bank account](https://www.moneysupermarket.com/current-accounts/digital-banking/) (you can do this before you arrive in the UK, and you will not need a UK address to do so)
+* [find somewhere to live](https://www.gov.uk/government/publications/how-to-rent)
+* [get a national insurance (NI) number](https://www.gov.uk/national-insurance/your-national-insurance-number)
+* [register with a doctor](https://www.nhs.uk/nhs-services/gps/how-to-register-with-a-gp-surgery/)
+* [find childcare](https://www.gov.uk/get-childcare)
+* [learn about tax](https://www.gov.uk/income-tax)
+* learn about [indefinite leave to remain](https://www.gov.uk/indefinite-leave-to-remain)
 
-Schools have some flexibility on [teachers' salaries](/salaries-and-benefits). It varies by region and your level of skill, experience and qualifications.
 
-There are [specific salary requirements for a skilled worker visa](https://www.gov.uk/government/publications/teach-in-england-if-you-qualified-outside-the-uk/teach-in-england-if-you-qualified-outside-the-uk#visas-and-immigration).
 
 ## Learn about life as a teacher in England
 
+If you’ve successfully used the [Apply for QTS in England](https://apply-for-qts-in-england.education.gov.uk/eligibility/start) service, and have less than 2 years’ teaching experience, you can get valuable support and mentoring through the [early career framework](https://www.gov.uk/government/publications/early-career-framework). 
```

Comment:
Suggestion:
If you’ve successfully applied for QTS in England, and have less than 2 years’ teaching experience, you can get valuable support and mentoring through the [early career framework](https://www.gov.uk/government/publications/early-career-framework). Could this not link to GiT?

Reply:
Yes, didn't realise you covered it